docs/products/opensearch/howto/import-opensearch-data-elasticsearch-dump-to-aws.rst

```diff
@@ -0,0 +1,63 @@
+Copy data from Aiven for OpenSearch® to AWS S3 using ``elasticsearch-dump``
+===================================================
+
+It is a good practice to perform backups of your OpenSearch® data to another storage service. This way you can access your data and restore it if something unexpected happens to it. 
+
+In this article, you can find out how to dump your OpenSearch data to AWS S3 bucket.
+
+To copy the index data, we will be using ``elasticsearch-dump`` `tool <https://github.com/elasticsearch-dump/elasticsearch-dump>`__. You can read the `instructions on GitHub <https://github.com/elasticsearch-dump/elasticsearch-dump/blob/master/README.md>`_ on how to install it. From this library, we will use ``elasticdump`` command to copy the input index data to an specific output. 
+
+.. note::
+
+    Make sure to have ``elasticsearch-dump`` `tool <https://github.com/elasticsearch-dump/elasticsearch-dump>`__ installed for the next steps.
```

Comment:
Would it make sense to move this note into as a prerequisite? 1st in the bullet list?

Reply:
sure, doing the same in the other file :-)